### PR TITLE
[Tests-Only] refactor share expire tests to not use skeleton files

### PR DESCRIPTION
**Pull request by dpakach**
## Description

Remove the use of skeleton files in tests for 
 webUISharingInternalUsersBlacklisted
 webUISharingInternalUsersExpire
 webUISharingInternalUsersExpireToRoot

## Related Issue

- Related https://github.com/owncloud/QA/issues/659

## Motivation and Context

To speed up ci

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

File: tests/acceptance/features/webUISharingInternalUsersExpire/shareWithUsersExpiringShares.feature [hunk 1]
```diff
@@ -69,6 +73,8 @@ Feature: Sharing files and folders with internal users with expiry date
     Given the setting "shareapi_default_expire_date_user_share" of app "core" has been set to "yes"
     And the setting "shareapi_enforce_expire_date_user_share" of app "core" has been set to "yes"
     And the setting "shareapi_expire_after_n_days_user_share" of app "core" has been set to "5"
+    Given user "Alice" has uploaded file "lorem.txt" to "lorem.txt"
```
Comment (Talank) [inline tests/acceptance/features/webUISharingInternalUsersExpire/shareWithUsersExpiringShares.feature:76]:
```suggestion
    And user "Alice" has uploaded file "lorem.txt" to "lorem.txt"
```

File: tests/acceptance/features/webUISharingInternalUsersExpire/shareWithUsersExpiringShares.feature [hunk 2]
```diff
@@ -83,6 +89,8 @@ Feature: Sharing files and folders with internal users with expiry date
...
```
Comment (Talank) [inline tests/acceptance/features/webUISharingInternalUsersExpire/shareWithUsersExpiringShares.feature:92]:
```suggestion
    And user "Alice" has uploaded file "lorem.txt" to "lorem.txt"
```

File: tests/acceptance/features/webUISharingInternalUsersExpire/shareWithUsersExpiringShares.feature [hunk 3]
```diff
@@ -101,6 +109,7 @@ Feature: Sharing files and folders with internal users with expiry date
     Given the setting "shareapi_default_expire_date_user_share" of app "core" has been set to "yes"
     And the setting "shareapi_enforce_expire_date_user_share" of app "core" has been set to "yes"
     And the setting "shareapi_expire_after_n_days_user_share" of app "core" has been set to "30"
+    Given user "Alice" has uploaded file "lorem.txt" to "lorem.txt"
```
Comment (Talank) [inline tests/acceptance/features/webUISharingInternalUsersExpire/shareWithUsersExpiringShares.feature:112]:
```suggestion
    And user "Alice" has uploaded file "lorem.txt" to "lorem.txt"
```

File: tests/acceptance/features/webUISharingInternalUsersExpire/shareWithUsersExpiringShares.feature [hunk 4]
```diff
@@ -126,6 +135,7 @@ Feature: Sharing files and folders with internal users with expiry date
...
```
Comment (Talank) [inline tests/acceptance/features/webUISharingInternalUsersExpire/shareWithUsersExpiringShares.feature:138]:
```suggestion
    And user "Alice" has uploaded file "lorem.txt" to "lorem.txt"
```

File: tests/acceptance/features/webUISharingInternalUsersExpire/shareWithUsersExpiringShares.feature [hunk 5]
```diff
@@ -151,6 +161,8 @@ Feature: Sharing files and folders with internal users with expiry date
     And the setting "shareapi_enforce_expire_date_group_share" of app "core" has been set to "yes"
     And the setting "shareapi_expire_after_n_days_user_share" of app "core" has been set to "10"
     And the setting "shareapi_expire_after_n_days_group_share" of app "core" has been set to "5"
+    Given user "Alice" has uploaded file "lorem.txt" to "lorem.txt"
```
Comment (Talank) [inline tests/acceptance/features/webUISharingInternalUsersExpire/shareWithUsersExpiringShares.feature:164]:
```suggestion
    And user "Alice" has uploaded file "lorem.txt" to "lorem.txt"
```

File: tests/acceptance/features/webUISharingInternalUsersExpire/shareWithUsersExpiringShares.feature [hunk 6]
```diff
@@ -177,6 +189,8 @@ Feature: Sharing files and folders with internal users with expiry date
     And the setting "shareapi_enforce_expire_date_group_share" of app "core" has been set to "yes"
     And the setting "shareapi_expire_after_n_days_user_share" of app "core" has been set to "5"
     And the setting "shareapi_expire_after_n_days_group_share" of app "core" has been set to "10"
+    Given user "Alice" has uploaded file "lorem.txt" to "lorem.txt"
```
Comment (Talank) [inline tests/acceptance/features/webUISharingInternalUsersExpire/shareWithUsersExpiringShares.feature:192]: same here

File: tests/acceptance/features/webUISharingInternalUsersExpire/shareWithUsersExpiringShares.feature [hunk 7]
```diff
@@ -197,6 +211,7 @@ Feature: Sharing files and folders with internal users with expiry date
   Scenario: change existing expiration date of an existing share to a date beyond the enforced maximum expiry date
     Given the setting "shareapi_default_expire_date_user_share" of app "core" has been set to "yes"
     And the setting "shareapi_expire_after_n_days_user_share" of app "core" has been set to "7"
+    Given user "Alice" has uploaded file "lorem.txt" to "lorem.txt"
```
Comment (Talank) [inline tests/acceptance/features/webUISharingInternalUsersExpire/shareWithUsersExpiringShares.feature:214]: and here

File: tests/acceptance/features/webUISharingInternalUsersExpireToRoot/shareWithUsersExpiringShares.feature [hunk 8]
```diff
@@ -46,6 +49,7 @@ Feature: Sharing files and folders with internal users with expiry date
     Given the setting "shareapi_default_expire_date_user_share" of app "core" has been set to "yes"
     And the setting "shareapi_enforce_expire_date_user_share" of app "core" has been set to "yes"
     And the setting "shareapi_expire_after_n_days_user_share" of app "core" has been set to "5"
+    Given user "Alice" has uploaded file "lorem.txt" to "lorem.txt"
```
Comment (Talank) [inline tests/acceptance/features/webUISharingInternalUsersExpireToRoot/shareWithUsersExpiringShares.feature:52]: and here

File: tests/acceptance/features/webUISharingInternalUsersExpireToRoot/shareWithUsersExpiringShares.feature [hunk 9]
```diff
@@ -61,6 +65,8 @@ Feature: Sharing files and folders with internal users with expiry date
...
```
Comment (Talank) [inline tests/acceptance/features/webUISharingInternalUsersExpireToRoot/shareWithUsersExpiringShares.feature:68]: and here

**Review by Talank — CHANGES_REQUESTED** [review]
Use And for consecutive Givens

**Comment by dpakach** [comment]
@Talank fixed everything, there was bug in code

**Review by Talank — APPROVED** [review]

**Review by individual-it — APPROVED** [review]

**Pull request merged**